spell/psm-checklists.md

```diff
@@ -0,0 +1,137 @@
+- Destination PSM:
+    - [ ] ilk matches destination PSM
+    - [ ] IF present, `tin` matches the value in the Exec Sheet
+    - [ ] IF present, `tout` matches the value in the Exec Sheet
+    - [ ] `AutoLine` is updated according to the Exec Sheet
+    - [ ] `buf` matches the value in the Exec Sheet
+    - [ ] IF required (`rush() > 0`), `fill` is called to ensure liquidity is immediately available after spell execution
+- Test coverage:
+    - [ ] Collateral amount that is fetched from the source PSM matches the collateral is added to the destination PSM
+        - [ ] `vat.urns(DST_ILK, dstPsm).art` is increased **at least** by the moved amount
+        - [ ] `gem` balance of `dstPocket` is increased **exactly** by the moved amount
+        - [ ] Dai balance of `dstPsm` is equal to `dstBuf`, UNLESS limited by the debt ceiling (`line`)
+        - [ ] `vat.urns(SRC_ILK, srcPsm).art` is decreased **exactly** by the moved amount
+        - [ ] `vat.urns(SRC_ILK, srcPsm).ink` is decreased **exactly** by the moved amount
+        - [ ] `gem` balance of `srcPsm.gemJoin()` is decreased **exactly** by the moved amount
+    - [ ] No bad debt is left behind
+        - [ ] `vat.vice()` is unchanged after the spell execution
+        - [ ] `vat.sin(MCD_PAUSE_PROXY)` is unchanged after the spell execution
+
+## LitePSM On-boarding Checklist
+
+- Deployed Contracts
+    - `DssLitePsm`
+        - [ ] Optimizer is **enabled**
+        - [ ] Contract has been reviewed by minimum 2 specialized [Active Ecosystem Actors](https://mips.makerdao.com/mips/details/MIP101#2-8-2-active-ecosystem-actors)
+        - [ ] Contract has been audited by an external party
+            - [ ] Audit reports are public
+            - [ ] All issues found were fixed or acknowledged
+            - [ ] No code changes are done after final Audit report
+        - [ ] Deployed contract matches the code in the [repo](https://github.com/makerdao/dss-lite-psm/)
+        - Constructor params:
+            - [ ] ilk is named `LITE-PSM-{TOKEN_SYMBOL}-A` (i.e. `LITE-PSM-USDC-A`)
+            - [ ] `gem` matches the expected token address
+            - [ ] `daiJoin` matches `MCD_JOIN_DAI` from the chainlog
+            - [ ] `pocket` matches the pocket address in the Exec Sheet
+        - [ ] Deployer no longer has privileged access (`wards(deployer) == 0`)
+        - [ ] `MCD_PAUSE_PROXY` has been authed (i.e. `require(WardsLike(LITE_PSM).wards(MCD_PAUSE_PROXY) == 1)`)
+        - [ ] Sanity check: `DssLitePsm` has `type(uint256).max` approval to spend `gem` on behalf of `pocket`
+    - `DssLitePsmMom`
+        - [ ] Optimizer is **enabled**
+        - [ ] Contract has been reviewed by minimum 2 specialized [Active Ecosystem Actors](https://mips.makerdao.com/mips/details/MIP101#2-8-2-active-ecosystem-actors)
+        - [ ] Contract has been audited by an external party
+            - [ ] Audit reports are published in the [repo](https://github.com/makerdao/dss-lite-psm/)
```

Comment:
As above, let's not trust the PDFs found along the code. See another proposal [in this thread](https://github.com/makerdao/pe-checklists/pull/34/files#r1684592531)